tox.ini

```diff
@@ -24,6 +24,7 @@ setenv =
 
     c44: _DEP_COVERAGE=coverage==4.4.2
     c45: _DEP_COVERAGE=coverage==4.5.3
+    c50: _DEP_COVERAGE=coverage==5.0a6
```

Comment:
Also add it to envlist and Travis then.
(https://github.com/pytest-dev/pytest-cov/pull/291)

Reply:
I'm not sure what the point of envlist is here.  It's the default if no "-e" is specified, but I've been told, "oh, no one runs all the environments."  Maybe there's a use of envlist I don't understand?

Reply:
Hmm, ci/bootstrap.py.  How broad a test matrix do we want at the end of this?

Reply:
`envlist` can also be seen as a list of valid/supported envs, displayed with `tox -l`.

> Hmm, ci/bootstrap.py. How broad a test matrix do we want at the end of this?

￼I personally find it too broad already.
Makes sense to only enable it explicitly while working on this PR.

Reply:
`./ci/bootstrap.py` needs to be run after editing the envlist section.  Travis can support 200 items (maybe) in the matrix, Appveyor less (but we have some grouping there).

I think it's fine if we run both 4.5 and 5.0 for a while (pytest devs will forgive us for taking travis CI time 😇)